recipes/elfio/all/conanfile.py

```diff
@@ -1,32 +1,35 @@
+from conan import ConanFile
+from conan.tools.layout import basic_layout
+from conan.tools.files import get, copy, rename
+from conan.tools.build import check_min_cppstd
 import os
-from conans import ConanFile, tools
 
 
 class ElfioConan(ConanFile):
     name = "elfio"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "http://elfio.sourceforge.net"
     description = "A header-only C++ library that provides a simple interface for reading and generating files in ELF binary format."
-    topics = ("conan", "elfio", "elf")
+    topics = ("elfio", "elf")
     license = "MIT"
     settings = "compiler"
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def configure(self):
         if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, 11)
+            check_min_cppstd(self, 14)
```

Comment:
Does it require C++14 since 3.11? If yes, it would need a condition, based on the package version.

Reply:
> Does it require C++14 since 3.11?